pytorch_lightning/trainer/trainer.py

```diff
@@ -376,12 +378,20 @@ def init_optimizers(self, optimizers):
         # two lists
         elif len(optimizers) == 2 and isinstance(optimizers[0], list):
             optimizers, lr_schedulers = optimizers
+            lr_schedulers, self.reduce_lr_on_plateau_scheduler = self.configure_schedulers(lr_schedulers)
             return optimizers, lr_schedulers
 
         # single list or tuple
         elif isinstance(optimizers, list) or isinstance(optimizers, tuple):
             return optimizers, []
 
+    def configure_schedulers(self, schedulers):
+        for i in range(len(schedulers)):
```

Comment:
consider `for i, scheduler for enumerate(schedulers):`

Reply:
@Borda maybe make the changes for the author to get this PR in quickly?

Reply:
@williamFalcon seems we dont have permissions to edit directly..

Reply:
correct, only maintainers and owners can do these changes (also if the PR authors click allow changes, which is here true by default)